cms/static/js/spec/views/active_video_upload_list_spec.js

```diff
@@ -59,6 +59,31 @@ define(
                 });
             };
 
+            describe('supported file formats', function() {
+                it('should not show unsupported file format notification for supported files', function() {
+                    var supportedFiles = {
+                        files: [
+                            {name: 'test-1.mp4', size: 0},
+                            {name: 'test-1.mov', size: 0}
+                        ]
+                    };
+                    this.view.$uploadForm.fileupload('add', supportedFiles);
+                    expect(this.view.fileErrorMsg).toBeNull();
```

Comment:
Instead of checking `fileErrorMsg` for `null`, i think it would be better to explicitly check the notificationview visibility. I think we should remove `fileErrorMsg` from view as i mentioned earlier.